CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+Contributions are **welcome** and will be fully **credited**. We accept contributions via pull requests on GitHub.
+
+## Pull Requests
+
+- **[PSR-2 Coding Standard.](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)** The easiest way to apply the conventions is to install [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer).
+- **Ask before working on a feature.** Before spending hours on a feature, open an issue and let's discuss about it.
+- **Add tests!** Your patch won't be accepted if it doesn't have tests.
+- **Document any change in behaviour.** Make sure the `README.md` and any other relevant documentation are kept up-to-date.
+- **Create feature branches.** Don't ask us to pull from your master branch.
```

Comment:
Don't ask us to pull from your master branch.

The tone is not very welcoming. I suggest:

> Small fixes can be pulled from your master branch, but features needs to be pulled from a feature branch.